packages/opentelemetry-plugin-http/package.json

```diff
@@ -55,7 +55,7 @@
     "@types/superagent": "4.1.7",
     "axios": "0.19.2",
     "codecov": "3.7.0",
-    "eslint": "6.8.0",
+    "eslint": "7.2.0",
```

Comment:
Why do need `eslint` here? @dyladan any idea?

Reply:
Not sure. It's probably not needed.